scripts/update-rules.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const recommendedRulesFile = require('../lib/config/recommended');
+const octaneRulesFile = require('../lib/config/octane');
+const stylisticRulesFile = require('../lib/config/stylistic');
+const root = require('../lib/rules');
```

Comment:
Could this be `rules` instead of `root`?

Reply:
Fixed!